src/Uno.UI/UI/Xaml/Media/ImageSource.Android.cs

```diff
@@ -18,8 +18,6 @@ namespace Windows.UI.Xaml.Media
 {
 	public partial class ImageSource
 	{
-		private const string DrawableUriPrefix = "drawable://";
```

Comment:
@MartinZikmund it this something you know about?

Reply:
I am guessing this is not needed anymore 🤔